src/pages/signin/LoginForm/BaseLoginForm.tsx

```diff
@@ -268,6 +268,7 @@ function BaseLoginForm({account, credentials, closeAccount, blurOnSubmit = false
                     onSubmitEditing={validateAndSubmitForm}
                     autoCapitalize="none"
                     autoCorrect={false}
+                    autoFocus
```

Comment:
Adding autofocus props caused [this bug](https://github.com/Expensify/App/issues/43806)